README.md

```diff
@@ -37,20 +37,25 @@ A [Terraform](https://www.terraform.io) 0.12 module to create a scalable and hig
 This module offers a convenient way to create Route53 zones and records.
 
 - **Zones**:
-  You can either create a single zone by passing a string (e.G. `"mineiros.com"`)
-  or multiple zones by passing a list of strings (e.G. `["mineiros.io", "mineiros.com]`)
-  as the `name` parameter. `name = "mineiros.io"`. You can also share the same
-  records among multiple zones. Please see the [example][same-records-example] for details.
+  You can either create a single zone by passing a string (e.G. `"mineiros.com"`) or multiple zones by passing a list
+  of strings (e.G. `["mineiros.io", "mineiros.com]`) as the `name` parameter.
+  `name = "mineiros.io"`. You can also share the same records among multiple zones. Please see the
+  [example](https://github.com/mineiros-io/terraform-aws-route53/tree/master/examples/multiple-domains-same-records)
+  for details.
 
 - **Records**:
-  Records can be shared among zones or be defined for a single zone only.
-  We support [alias], [weighted] and [failover] records.
+  Records can be shared among zones or be defined for a single zone only. We support
+  [alias](https://aws.amazon.com/premiumsupport/knowledge-center/route-53-create-alias-records/),
+  [weighted](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/TutorialLBRMultipleEC2InRegion.html)
+  and [failover](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/dns-failover-configuring.html)
+  records.
```

Comment:
seems to revert already applied improvements.

Reply:
resolved in ec761c4